rabbitmq/consumer.go

```diff
@@ -0,0 +1,113 @@
+package rabbitmq
+
+import (
+	"github.com/palantir/stacktrace"
+
+	"go.uber.org/zap"
+
+	"github.com/sumup-oss/go-pkgs/logger"
+
+	"github.com/streadway/amqp"
+)
+
+func NewRabbitMQConsumer(
```

Comment:
Also style nitpick, but don't put constructors before type struct declarations. It's weird and most of the time more confusing than beneficial to find something